src/pyhf/simplemodels.py

```diff
@@ -68,3 +68,70 @@ def hepdata_like(signal_data, bkg_data, bkg_uncerts, batch_size=None):
         ]
     }
     return Model(spec, batch_size=batch_size)
+
+
+def correlated_background(signal, bkg, bkg_up, bkg_down, batch_size=None):
+    r"""
+    Construct a simple single channel :class:`~pyhf.pdf.Model` with a
+    :class:`~pyhf.modifiers.histosys` modifier representing a two component
+    correlated background uncertainty.
+
+    Args:
+        signal (:obj:`list`): The data in the signal sample.
+        bkg (:obj:`list`): The data in the background sample.
+        bkg_up (:obj:`list`): The background sample under a :math:`+1\sigma` variation.
+        bkg_down (:obj:`list`): The background sample under a :math:`-1\sigma` variation.
+        batch_size (:obj:`None` or :obj:`int`): Number of simultaneous (batched) Models to compute.
+
+    Returns:
+        ~pyhf.pdf.Model: The statistical model adhering to the :obj:`model.json` schema.
+
+    Example:
+        >>> import pyhf
+        >>> pyhf.set_backend("numpy")
+        >>> model = pyhf.simplemodels.correlated_background(
+        ...     signal=[12.0, 11.0],
+        ...     bkg=[50.0, 52.0],
+        ...     bkg_up=[53.0, 59.0],
+        ...     bkg_down=[45.0, 49.0],
```

Comment:
In the practical applications I am aware of, `histosys` modifiers are used for pure shape variations, and the total yield of the two histograms in the modifier would be the same. The normalization effects can then be split out to a (correlated) `normsys` modifier to have an effective log-normal constraint and protect against negative yields.

Doing this kind of split automatically may be a bit too much for `simplemodels` and lead to confusion. If the aim of #1434 is the use of more standard LHC-like models, it might be worth considering.

Reply:
Very good point @alexander-held. I'll fix this in the morning my time, as it appears that my brain is mush enough that I'm no longer thinking about what I'm typing. :grimacing:

Reply:
yes i'd do

```
        ...     bkg=[50.0, 50.0],
        ...     bkg_up=[45.0, 55.0],
        ...     bkg_down=[55.0, 45.0],
```

or sth similar

Reply:
Here is an example for the difference this can make:
```python
model = correlated_background(
    signal=[12.0, 11.0], bkg=[50.0, 52.0], bkg_up=[73.0, 79.0], bkg_down=[25.0, 29.0]
)
print("current model:")
print(f"  nominal:  {model.expected_data([1.0, 0.0], include_auxdata=False)}")
print(f"  -1 sigma: {model.expected_data([1.0, -1.0], include_auxdata=False)}")
print(f"  -3 sigma: {model.expected_data([1.0, -3.0], include_auxdata=False)}")


print("new model:")
spec_new = {
    "channels": [
        {
            "name": "single_channel",
            "samples": [
                {
                    "name": "signal",
                    "data": [12.0, 11.0],
                    "modifiers": [{"name": "mu", "type": "normfactor", "data": None}],
                },
                {
                    "name": "background",
                    "data": [50.0, 52.0],
                    "modifiers": [
                        {
                            "name": "correlated_bkg_uncertainty",
                            "type": "histosys",
                            "data": {
                                "hi_data": [48.99, 53.01],
                                "lo_data": [47.22, 54.78],
                            },
                        },
                        {
                            "name": "correlated_bkg_uncertainty",
                            "type": "normsys",
                            "data": {"hi": 1.49, "lo": 0.53},
                        },
                    ],
                },
            ],
        }
    ]
}
model_new = pyhf.Model(spec_new)
print(f"  nominal:  {model_new.expected_data([1.0, 0.0], include_auxdata=False)}")
print(f"  -1 sigma: {model_new.expected_data([1.0, -1.0], include_auxdata=False)}")
print(f"  -3 sigma: {model_new.expected_data([1.0, -3.0], include_auxdata=False)}")
```
resulting in
```
current model:
  nominal:  [62. 63.]
  -1 sigma: [37. 40.]
  -3 sigma: [-13.  -6.]
new model:
  nominal:  [62. 63.]
  -1 sigma: [37.0266 40.0334]
  -3 sigma: [18.20221582 19.98323818]
```
where the version with the histosys/normsys split has the built-in protection against negative yields.

Reply:
The split does not provide complete safety either, if the shapes are very large then it is possible to still have negative yield predictions in bins with sufficiently strong pulls.

Reply:
This probably should exist in a learn notebook, rather than making the simplemodel more explicit. One could certainly add a `normalized=True` kwarg that could additionally split the bkg into a shape and normalization.

Reply:
I agree we should have docs on this, but maybe we keep this trivially simple - we shouldn't necessarily be in the business of buliding production grade models for users . These simple models are mostly for quick&dirty sensitivity tests.